drivers/Makefile.include

```diff
@@ -84,6 +84,10 @@ ifneq (,$(filter ccs811,$(USEMODULE)))
   USEMODULE_INCLUDES += $(RIOTBASE)/drivers/ccs811/include
 endif
 
+ifneq (,$(filter cst816s,$(USEMODULE)))
+  USEMODULE_INCLUDES += $(RIOTBASE)/drivers/cst816s/include
+endif
+
```

Comment:
Now each driver add its own Makefile.include. Here it would have the following content:

```mk
USEMODULE_INCLUDES_cst816s := $(LAST_MAKEFILEDIR)/include
USEMODULE_INCLUDES += $(USEMODULE_INCLUDES_cst816s)
```

You can also remove the change in drivers/Makefile.include:
```suggestion
```